src/user.rs

```diff
@@ -306,24 +306,23 @@ impl User<Synching> {
         ));
         let instant = Instant::now();
 
-        let room = client
-            .get_joined_room(room_id)
-            .expect("Room must be Joined at this point");
-        let response = room.send(content, None).await;
-        match response {
-            Ok(response) => {
-                self.send(Event::RequestDuration((
-                    UserRequest::SendMessage,
-                    instant.elapsed(),
-                )))
-                .await;
-
-                self.send(Event::MessageSent(response.event_id.to_string()))
+        if let Some(room) = client.get_joined_room(room_id) {
```

Comment:
Let's add a `TODO` here for the future since we need to review the issue of having users not joined in the rooms.

Reply:
added on next incoming PR 👍